### PR TITLE
luci-app-guest-wifi: code alignment

### PR DESCRIPTION
**Pull request by Beginner-Go**

### CODE REVIEW

File: applications/luci-app-guest-wifi/luasrc/controller/guest-wifi.lua [hunk 1]
```diff
@@ -5,9 +5,8 @@ function index()
 		return
 	end
 
-	local page = entry({"admin", "network", "guest-wifi"}, cbi("guest-wifi"), translate("Guest-wifi"), 19)
-	page.i18n = "guest-wifi"
+	local page = entry({"admin", "network", "guest-wifi"}, cbi("guest-wifi"), _("Guest-wifi"))
+	page.order = 19
```
Comment (1715173329) [inline applications/luci-app-guest-wifi/luasrc/controller/guest-wifi.lua:9]: Why?

File: applications/luci-app-guest-wifi/root/etc/uci-defaults/luci-guest-wifi [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh 
+
+uci -q batch <<-EOF >/dev/null
```
Comment (1715173329) [inline applications/luci-app-guest-wifi/root/etc/uci-defaults/luci-guest-wifi:3]: Again don't rename ucis.

File: applications/luci-app-guest-wifi/luasrc/model/cbi/guest-wifi.lua [hunk 3]
```diff
@@ -3,64 +3,76 @@ require("luci.tools.webadmin")
...
 m = Map("guest-wifi", translate("Guest-wifi"))
 
-s = m:section(TypedSection, "guest-wifi", translate("Config"), translate("You can set guest wifi here. The wifi will be disconnected when enabling/disabling. When modifying the password, first disable the guest wifi, and then do the modification, save and apply. Finally check both Enable and Create, save and apply."))
+s = m:section(TypedSection, "guest-wifi", translate("Config"))
+s.description = translate("You can set guest wifi here. The wifi will be disconnected when enabling/disabling. When modifying the password, first disable the guest wifi, and then do the modification, save and apply. Finally check both Enable and Create, save and apply.")
 s.anonymous = true 
 s.addremove = false
 
-enable = s:option(Flag, "enable", translate("Enable"), translate("Enable or disable guest wifi"))
+enable = s:option(Flag, "enable", translate("Enable"))
```
Comment (1715173329) [inline applications/luci-app-guest-wifi/luasrc/model/cbi/guest-wifi.lua:11]: Use `o = s...` instead.

**Review by 1715173329 — CHANGES_REQUESTED** [review]

**Comment by Beginner-Go** [comment]
changes completed

File: applications/luci-app-guest-wifi/root/etc/uci-defaults/luci-guest-wifi (hunk 2 above)
Comment (Beginner-Go) [reply applications/luci-app-guest-wifi/root/etc/uci-defaults/luci-guest-wifi:3]: Does not affect operation

**Pull request closed without merging**